### DISCUSSION TITLE
Support for sp_getapplock with down migrations

### DISCUSSION THREAD

**Discussion by all4Git** [fluentmigrator/fluentmigrator#1661] [Ideas]
Hi,

I use your Database-Dependent Application Locking solution (https://fluentmigrator.github.io/articles/faq.html#database-dependent-application-locking) to run migrations safely from multiple servers.
As explained in the FAQ, your solution works for Up migrations, but is not implemented for Down migrations.
Would it be possible to implement this solution for Down migrations, and if so, is it a feature which could be added in an upcoming FluentMigrator release ?

Thanks.

**Comment by jzabroski** [comment]
I'm confused by this question.  The approach is defined for Up as an example.  You can certainly use this for Down migrations, but I would like to know more about what you want to accomplish in case there is something easier you can do.
